src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue

```diff
@@ -284,6 +326,50 @@ export default {
       }
     });
 
+    // calculate fees
+    const totalFee = computed(() => {
+      // if multisig
+      if(isMultiSig(selectedAccAdd.value)){
+        return parseFloat(lockFundTotalFee.value) + rentalFee.value + transactionFeeExact.value;
+      }else{
+        return rentalFee.value + transactionFeeExact.value;
+      }
+    });
+
+    watch(totalFee, (n) => {
+      if(balance.value < n && !isNotCosigner.value){
+        showNoBalance.value = true;
+        disabledMutableCheck.value = true;
+        disabledTransferableCheck.value = true;
+        disabledPassword.value = true;
+        disabledSupply.value = true;
+        disabledDivisibility.value = true;
+      }else{
+        showNoBalance.value = false;
+        disabledMutableCheck.value = false;
+        disabledTransferableCheck.value = false;
+        disabledPassword.value = false;
+        disabledSupply.value = false;
+        disabledDivisibility.value = false;
+      }
+    });
+
+    watch(isNotCosigner, (n) => {
+      if(n){
+        disabledMutableCheck.value = true;
+        disabledTransferableCheck.value = true;
+        disabledPassword.value = true;
+        disabledSupply.value = true;
+        disabledDivisibility.value = true;
+      }else{
+        disabledMutableCheck.value = false;
+        disabledTransferableCheck.value = false;
+        disabledPassword.value = false;
+        disabledSupply.value = false;
+        disabledDivisibility.value = false;
+      }
+    });
+
     const createMosaic = () => {
       let createStatus = mosaicTransaction(divisibility.value, supply.value, duration.value, durationOption.value, isMutable.value, isTransferable.value, walletPassword.value, selectedAccName.value, appStore, siriusStore);
```

Comment:
this still using ? there are using appStore and siriusStore

Reply:
Ops. This is no longer needed. To be replaced with create asset method once it is ready

Reply:
ok, maybe put some comment on it

Reply:
Okay noted